public/stylesheets/styles.css

```
@@ -5,3 +5,72 @@ dark-blue: #0a2c42;
 grey: #bfbfbf;
 font-family: Arial, Helvetica, sans-serif;
 */
+
+body {
+    background-color: lightgrey;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.index {
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.beer {
+    width: 25%;
+    height: 25%;
+    margin: 15px;
+}
+
+.btn {
+    margin:15px
+}
+
+.footer {
+    margin: 0;
+    display: flex;
+    align-items: center;
+    background-color: #0a2c42;
+    padding: 10px 15px 10px 15px;
+    color: white;
+}
+
+.wrap-container {
+    display: flex;
```

Aqui te recomiendo darle un alto a este div para que tu footer no quede en medio de la pagina despues lo que puedes hacer es restarle a todo ese vh el alto de tu footer y nav 60 y 56 😉 
```suggestion
    height: calc(100vh - 116px);
    display: flex;
```

---

Te recomiendo hacer lo mismo para tu vista de home ya que al igual queda en medio el footer